openapi/openapi/src/main/scala/endpoints4s/openapi/model/OpenApi.scala

```diff
@@ -1524,9 +1524,10 @@ object Schema {
 
   val simpleUUID = Primitive("string", format = Some("uuid"), None, None, None)
   val simpleString = Primitive("string", None, None, None, None)
-  val simpleInteger = Primitive("integer", None, None, None, None)
+  val simpleInt = Primitive("integer", format = Some("int32"), None, None, None)
```

Comment:
This breaks the binary compatibility, do you mind keeping the previous name, `simpleInteger`? (same for `simpleNumber`)

Reply:
Oh, right.
Do you prefer keeping the old format-less values just for binary compatibility?
Or actually use `simpleInteger` with `int32` format and `simpleNumber` with `double` format?

Reply:
I would just keep the previous names:

```suggestion
  val simpleInteger = Primitive("integer", format = Some("int32"), None, None, None)
```

Reply:
Binary compatibility should be fixed now. @julienrf  Sorry for the glitch!

Reply:
_I_ am sorry. Managing the compatibility is complex. In this case, I thought we could relax the compatibility guarantees to `BinaryCompatible` only (instead of `BinaryAndSourceCompatibility`) instead of making `simpleLong` private. Let me try that.